packages/docs/docs/docs/zeptoken.md

```diff
@@ -0,0 +1,38 @@
+---
+id: zeptoken
+title: The ZEPToken
+sidebar_label: The ZEPToken
+---
+
+ZeppelinOS' native ZEP token is used to incentivize the creation and maintenance of a healthy ecosystem of EVM packages. It does so via a series of mechanisms that are detailed in the following sections of this article.
+
+## Compatibility standards
+
+ZEP is an ERC20 token, controlled by a [TPL](https://github.com/TPL-protocol) (Transaction Permission Layer) jurisdiction. This allows regional jurisdictions to control which addresses can hold or exchange ZEP using KYC, or whatever other validation process are found to be adequate for the region.
+
+The ZEP token's source code can be found at [`zos/packages/vouching/contracts/ZEPToken.sol`](https://github.com/zeppelinos/zos/blob/master/packages/vouching/contracts/ZEPToken.sol).
+
+## EVM package vouching
+
+Anyone can create ZeppelinOS EVM packages for free. More precisely, you don't need to hold or spend ZEP to create an EVM package. 
+
+However, ZeppelinOS provides a mechanism in which an EVM package can be registered and vouched for using ZEP. The collected stake of an EVM package can be challenged by other ZEP holders whenever a deficiency in the EVM package is presented for evaluation. In such a situation, the package's stake could be slashed in favor of the challenger.
```

Comment:
I think @elopio had the goal to use `vouch` instead of `stake`, to avoid confusion. If so, we should change all mentions to stake in this doc.

Reply:
Yes, I am aware of the preference of `vouch` vs `stake`. However, I find it hard to put into sentences because the latter is quantitative and the former is qualitative.

Some examples:
"The collected stake of an EVM package" => "The collected vouch of an EVM package"
"A package's stake could be slashed..." => "A package's vouch could be slashed..."

Reply:
We could use:
- "The tokens vouched for an EVM Package"
- "The vouched tokens for a package could be slashed..."

But I'll defer that decision to you and @elopio, to remain consistent with the rest of the site.